File: tests/fixtures/java/ExcessArgs.java
public class ExcessArgs {
    public static String createUser(String first, String last, String email,
                                     String phone, String addr, String city,
                                     String state, String zip) {
        return first + " " + last;
    }

    public static int simpleFunc(int a, int b) {
        return a + b;
    }
}

class UserService {
    private String db;
    private String cache;
    private String logger;
    private String mailer;
    private String validator;
    private String scheduler;

    public UserService(String db, String cache, String logger,
                       String mailer, String validator, String scheduler,
                       String extra7, String extra8, String extra9) {
        this.db = db;
        this.cache = cache;
        this.logger = logger;
        this.mailer = mailer;
        this.validator = validator;
        this.scheduler = scheduler;
    }

    public String getUser(String userId) {
        return this.db;
    }
}

File: tests/fixtures/kotlin/ExcessArgs.kt
fun createUser(
    first: String,
    last: String,
    email: String,
    phone: String,
    addr: String,
    city: String,
    state: String,
    zip: String
): String {
    return "$first $last"
}

fun simpleFunc(a: Int, b: Int): Int {
    return a + b
}

class UserService(
    private val db: String,
    private val cache: String,
    private val logger: String,
    private val mailer: String,
    private val validator: String,
    private val scheduler: String,
    private val extra7: String,
    private val extra8: String,
    private val extra9: String
) {
    fun getUser(userId: String): String {
        return db
    }
}
